### ISSUE TITLE
Video Playback support

### ISSUE THREAD

**Issue by rumaan** [rumaan/nasa_apod_flutter#3]
Some responses in `url` have video

**Comment by lvlasuod** [comment]
hi, may I work on this?

**Comment by rumaan** [comment]
> hi, may I work on this?

Yeah sure. Just make sure it doesn't break the current implementation. So in the end the app should now show the APOD image as well as video if any.

Check out the API response for the date `2019-10-01`.
